python/lsst/obs/base/ingest.py

```diff
@@ -149,7 +149,8 @@ def makeTransferChoiceField(doc="How to transfer files (None for no transfer).",
                  "auto": "choice will depend on datastore",
                  "link": "hard link falling back to symbolic link",
                  "hardlink": "hard link",
-                 "symlink": "symbolic (soft) link"},
+                 "symlink": "symbolic (soft) link",
+                 "relsymlink": "relative symbolic link"},
```

Comment:
Consider the following style to make such changes diff a bit cleaner:
```suggestion
                 "relsymlink": "relative symbolic link",
        },
```